x-pack/plugins/index_lifecycle_management/__jest__/client_integration/edit_policy/edit_policy.test.ts

```diff
@@ -700,33 +700,60 @@ describe('<EditPolicy />', () => {
 
   describe('searchable snapshot', () => {
     describe('on cloud', () => {
-      beforeEach(async () => {
-        httpRequestsMockHelpers.setLoadPolicies([getDefaultHotPhasePolicy('my_policy')]);
-        httpRequestsMockHelpers.setListNodes({
-          isUsingDeprecatedDataRoleConfig: false,
-          nodesByAttributes: { test: ['123'] },
-          nodesByRoles: { data: ['123'] },
-        });
-        httpRequestsMockHelpers.setListSnapshotRepos({ repositories: ['found-snapshots'] });
+      describe('new policy', () => {
```

Comment:
I noticed the test file is called `edit_policy.test.ts`. I'm not familiar with all the test cases here, but perhaps it should be renamed if we're testing both new and existing policies here?

Reply:
This is a really good point. For now this lines up with the component/section name of the plugin. The semantics are not quite right, if you think it is worth it, would you mind opening an issue for this change? It seems like something that needs a bit of thought as to what it should be called and whether any restructuring/refactoring is required.